html/js/librenms.js

```diff
@@ -441,10 +417,10 @@ function destroy_map(id) {
 
 function populate_map_markers(map_id, group_radius = 10, status = [0,1], device_group = 0) {
```

Comment:
Lets move this to worldmap.blade.php (I think that is the only usage), then we can use the route() helper to get the ajax url.

Reply:
done